src/components/login/index.js

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
-
-// import grid from '../../../node_modules/flexboxgrid/dist/flexboxgrid.css';
-// import styles from './login.css';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import LoginForm from './loginForm';
+import styles from './login.css';
 
 
 /**
- * The container component containing login and create account functionality
+ * The container component containing login
+ * and create account functionality
  */
-class Login extends React.Component {
-  constructor() {
-    super();
-    this.title = 'Login';
-  }
-  render() {
-    return (
-      <h1>{this.title}</h1>
-    );
-  }
-}
-export default Login;
+const LoginComponent = () => (
+  <section>
+    <div className={`box ${styles.paddingTop}`}>
+      <div className={`${grid.row}`}>
+        <div className={grid['col-xs-2']}></div>
```

Comment:
Please remove two extra '<div className={grid['col-xs-2']}></div>' and add "grid[‘center-xs’]" class to your row. Because we use a flexbox based grid system we don't need those extra div any more.You can find more about it here : http://flexboxgrid.com/